Doxyfile

```diff
@@ -0,0 +1,18 @@
+DOXYFILE_ENCODING      = UTF-8
+PROJECT_NAME           = "tangle-accelerator"
+PROJECT_NUMBER         = 0.0.1
```

Comment:
The versioning should be mapped to #51 .

Reply:
Right, initial dev release should be 0.1.0 instead.